DayrSuryan/DSEthiop23.xml

```diff
@@ -27,50 +27,134 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                             ናይ፡ ነፍስናን፡ ናይ፡ መንፈስናን፡ ናይ፡ ሥጋናን፡ ቍስሊ፡ እትፍውሶ፡ ንስኻ፡ እኻ።</q>
+                     <incipit xml:lang="ti">ዎ፡ ንዓለም፡ ኵሉ፡ ዝኀዝካ፡ ጐይታና፡ እግዚአብሔር፡ አብ፡
+                             ናይ፡ ነፍስናን፡ ናይ፡ መንፈስናን፡ ናይ፡ ሥጋናን፡ ቍስሊ፡ እትፍውሶ፡ ንስኻ፡ እኻ።
                            </incipit>
-                              <explicit>
-                        <q xml:lang="ti">
+                     <explicit  xml:lang="ti">
                                 ብጐይታና፡ ብኢየሱስ፡ ክርስቶስ፡ ምስኡ፡ አብኡ፡ ዘሎ፡ ክብርን፡
                                  ስብሐትን፡ ጽንዓትን፡ ነአኻ፡ ይግባእ፡ ምስኡን፡ ምስ፡
                              መንፈስ፡ ቅዱስን፡ ሎምን፡ ንኵሉ፡ ጊዜን (ንወትሩ፡ መዓልትን፡) ንዘለዓለምን።
-                             ሕዝቢ፡ አሜን።</q>
+                             ሕዝቢ፡ አሜን።
                      </explicit>
                           </msItem>
 
-
                   <msItem xml:id="ms_i3">
-                     <title type="complete">Śǝrʿāta qǝddāse</title>
-                     <textLang mainLang="gez"/>
+                     <locus from="1r" to="9r"/>
+                     <title type="complete" ref="LIT5909PrayersTi"/>
+                     <textLang mainLang="ti"/>
+                     
+                     <incipit xml:lang="ti">
+                        <locus target="#1r"/>
+                        ብአቦ፡  ስም፡  ብወዲ፡  ስምን፡  ብመንፈስ፡  ቅዱስ፡  ስምን፡ 
+                           ሐደ፡  አምላክ፡  አሜን። በስመ፡  አብ፡  ወወልድ፡  ወመንፈስ፡  ቅዱስ፡   አሐዱ፡  አምላክ፡  አሜን ።
+                           እግዚኦ፡  ነመስግነከ፡  አሎና፡  እግዚኦ፡  ነክብረካ፡  አሎና፡  እግዚኦ፡  ብአኻ፡  ንእመን፡  አሎና።
+                     </incipit>
+                     
+                     <explicit  xml:lang="ti">
+                              <locus target="#9r"/>
+                           ከምቲ፡  ነቦታትና፡  ንአብርሃምን፡  ንዘርዑን፡  ዝነገሮም፡  ክሳዕ፡  ዘለዓለም፡  ምሕረቱ፡  እናዘከረ፡  
+                                    ንእስራኤል፡  ባርያኡ፡  ተቀበሎ። (ብሥራተ፡  ሉቃ፡  1፡  46-55።)
+                                    ካብ፡  እዙይ፡  ንደኀር፡  ዘሎ፡  ከአ፡  እቲ፡  ጸሎታት፡  እዩ። ሰለ፡  እዚ፡  ባህልን፡  እምነትን፡  ፍቅሪ፡  ዓድና፤ ሎሚ፡  ግዝእቲ፡  ጽባሕ፡  ሐራ፡  እትከውን፡  
+                                    ኤርትራ፡  ከይትርስዑ፡  ኢለ፡  እነሄ፡  ጽሐፈልኩም፡  አሎኩ፡  ኤርትራዊ፡  አቦኩምን፡  ሐውኩምን፡  አባ፡  ጴጥሮስ፡  ገብረ፡  ሥላሴ።
+                           </explicit>
+                                 
                   </msItem>
+                  
 
                   <msItem xml:id="ms_i4">
-                     <locus from="66r"/>
-                     <title type="complete">Nāy ʾamānǝyān qǝdolāse</title>
-                     <textLang mainLang="ti"/>
+                     <title type="complete" ref="LIT5910QeddaseTi"/>
+                     
+                     <msItem xml:id="ms_i4.1">
+                        <locus from="12r"/>
+                        <title type="complete">Foreword</title>
+                        <textLang mainLang="en" otherLangs="ti"/>
+                        <note>Foreword in English (printed)  and Tigrinya on the composition of the Tigrinya translation of the
+                           Śǝrʿāta qǝddāse.</note>
```

Comment:
maybe a bit more clear: "... translatio of the Prefatory Service (Ser'ata qeddase)"

Reply:
Sorry, Preparatory Service.